src/openrct2/ride/Ride.cpp

```diff
@@ -3551,14 +3631,19 @@ ResultWithMessage Ride::CreateVehicles(const CoordsXYE& element, bool isApplying
  * preceding that block.
  *  rct2: 0x006DDF9C
  */
-void Ride::MoveTrainsToBlockBrakes(TrackElement* firstBlock)
+void Ride::MoveTrainsToBlockBrakes(CoordsXYE* currentElement)
```

Comment:
id prefer this was a coordsxy and a trackelement. Coordsxye is a bit of a rubbish structure and we should limit its use.